app/controllers/editorial/submissions_controller.rb

```diff
@@ -10,7 +10,7 @@ class Editorial::SubmissionsController < Editorial::SectionsController
 
 
     def index
-      @submissions = scope.open.for(current_user)
+      @submissions = scope.open
```

Comment:
Would this potentially make the submissions index grow quite largely and useless? Particularly if not sorted in any meaningful way?